lib/resolvers/crud.resolver.ts

```diff
@@ -63,8 +63,25 @@ export function createResolver(definition: Definition): Provider {
       )
       input: any,
     ) {
-      input;
-      throw new graphql.GraphQLError('Not implemented yet');
+      const updated = await this.model.findOneAndUpdate({ _id: input.id }, input);
+      if (util.isNil(updated))
```

Comment:
We do support create recusive but we won't support update recursive so you can remove 67 -> 80.